gulpfile.js

```diff
@@ -0,0 +1,101 @@
+/* jshint node:true */
+const _ = require('underscore');
+const babel = require('rollup-plugin-babel');
+const Erik = require('erik');
+const gulp = require('gulp');
+const MultiBuild = require('multibuild');
+const multiEntry = require('rollup-plugin-multi-entry');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const rootImport = require('rollup-plugin-root-import');
+
+const SHOULD_WATCH = (process.env.WATCH === 'true');
+const SHOULD_BUILD_TESTS = (process.env.npm_lifecycle_event === 'test');
+
+const SRC_FILES = 'src/**/*';
+const TEST_FILES = ['spec/**/*.js', '!spec/tests.js'];
+
+const build = new MultiBuild({
+  gulp,
+  targets: _.compact([
+    'index',
+    SHOULD_BUILD_TESTS && 'tests'
+  ]),
+  entry: (target) => (target === 'tests') ? TEST_FILES : `src/${target}.js`,
+  rollupOptions: (target) => {
+    return {
+      external: ['jquery', 'backbone', 'underscore'],
```

Comment:
@chuym, you'll want to pull in the external rollup config [here](https://github.com/mixmaxhq/backbone-publication/blob/master/rollup.config.js) (or internalize it for the build of this module using a new `gulp` task on for the `npm` `build` script target).